cnd/src/network.rs

```diff
@@ -582,26 +600,39 @@ impl libp2p::swarm::NetworkBehaviourEventProcess<orderbook::BehaviourOutEvent> f
                 let storage = self.storage.clone();
                 let order_id = order.id;
 
+                let hbit = hbit::CreatedSwap {
+                    amount: partial_take_quantity,
+                    final_identity: final_identity.into(),
+                    network: order.bitcoin_ledger,
+                    absolute_expiry: order.bitcoin_absolute_expiry,
+                };
+
+                let ethereum_quantity = match order.ethereum_quantity(partial_take_quantity) {
```

Comment:
As a temporary measure, `token_quantity` might be a good name. We are only doing BTC-DAI but much of the code is still generic of the given ERC20 contract. "Ethereum quantity" is a bit too generic I think. What does that even refer to? :)